### ISSUE TITLE
Diff false positives for timestamptz

### ISSUE THREAD

**Issue by johnaschroeder** [pgmodeler/pgmodeler#1280]
**Bug description**
Diff false positives for timestamptz type.  Every timestamptz column shows up as a diff like the following:
```
ALTER TABLE myschema.mytable ALTER COLUMN date_created TYPE timestamptz;
```
Idempodent, but annoying as I have a lot of them and having them mixed in with valid changes makes it more difficult to validate a diff.  The database was created from pgmodeler and all the date columns are definitely timestamptz.

**How to reproduce**
Create a timestamptz type and run diff with the model as the source and the database as the target.  

**Expected behavior**
columns that are already timestamptz types should not appear in diff

**Info about your desktop**
 - OS: Mac OS X
 - Version: 10.13.6
 - Window manager:
 - pgModeler version: V0.9.1 build: 20180513
 - Qt version: Qt 5.9.3

**Additional info**
I combed through the issues, and found an old one from 2016 that mentioned a different false positive, but it looks like it was fixed.  I must be missing something, as this is a pretty common type.  Apologies if it's obvious, but wanted to open an issue if for no other reason than others might have the same problem.  Great tool, FYI we contributed by purchasing the binary.

**Comment by rkhaotix** [comment]
Hi @johnaschroeder 

Sorry the delay. This is an issue that happens on 0.9.1. Did you tried the 0.9.2-beta? 

I tested a small sample here and no more false-positives related to timestamp and other data types were generated.

**Comment by johnaschroeder** [comment]
@rkhaotix Yes, I upgraded to 0.9.2-beta build 20190531 and it still happens against postgres 10.

**Comment by rkhaotix** [comment]
Okay. So, could you provide an small sample model in which the problem occurs?

**Comment by johnaschroeder** [comment]
@rkhaotix Here is a sample model that duplicates the problem.  Create a database named 'test' then run the sql in test.sql to create a simple table with a date field.  Then open test.dbm and run a diff.  You should see the same result as in test-diff.sql: 
```
-- [ Changed objects ] --
ALTER TABLE test.new_table ALTER COLUMN date_created TYPE timestamptz;
-- ddl-end --
```
Hope that helps!
[test.zip](https://github.com/pgmodeler/pgmodeler/files/3299435/test.zip)

**Comment by andrea689** [comment]
resolved in v0.9.2-beta1
